CMakeLists.txt

```diff
@@ -37,6 +37,32 @@ option(OPENMC_USE_DAGMC       "Enable support for DAGMC (CAD) geometry"
 option(OPENMC_USE_LIBMESH     "Enable support for libMesh unstructured mesh tallies" OFF)
 option(OPENMC_USE_MPI         "Enable MPI"                                           OFF)
 
+# Warnings for deprecated options
+foreach(OLD_OPT IN ITEMS "openmp" "profile" "coverage" "dagmc" "libmesh")
+  if(DEFINED ${OLD_OPT})
+    string(TOUPPER ${OLD_OPT} OPT_UPPER)
+    message(WARNING "The OpenMC CMake option '${OLD_OPT}' has been deprecated. "
+      "Its value will be ignored. "
+      "Please use '-DOPENMC_USE_${OPT_UPPER}=${${OLD_OPT}}' instead.")
```

Comment:
The `profile` and `coverage` options have the form `-DOPENMC_ENABLE_...` rather than `-DOPENMC_USE_...`

Reply:
Ah so they do. Updating...